misk-admin/src/main/kotlin/misk/web/v2/DashboardPageBuilder.kt

```diff
@@ -0,0 +1,118 @@
+package misk.web.v2
+
+import kotlinx.html.TagConsumer
+import misk.config.AppName
+import misk.tailwinds.Link
+import misk.tailwinds.pages.MenuSection
+import misk.tailwinds.pages.Navbar
+import misk.turbo.buildHtml
+import misk.turbo.turbo_frame
+import misk.web.HttpCall
+import misk.web.dashboard.DashboardHomeUrl
+import misk.web.dashboard.DashboardNavbarItem
+import misk.web.dashboard.DashboardTab
+import wisp.deployment.Deployment
+import javax.inject.Inject
+
+/**
+ * Builds dashboard UI for index homepage.
+ */
+class DashboardPageBuilder @Inject constructor(
```

Comment:
this is a really weird class, especially "newBuilder"

Reply:
Open to renaming, the refactor came from realizing that there was a lot of code duplication across pages to build the dashboard nav UI. https://github.com/cashapp/misk/pull/2800/commits/9c817853121a6cd8deb8b819d3a300394ddeaa53

I'm a bit paranoid on accidental builder reuse after a gnarly SEV, could dial it back if it seems unreasonable.